contracts/SafeDrop.vy

```diff
@@ -0,0 +1,38 @@
+# @version 0.3.10
+
+merkle_root: public(bytes32)
+
+struct Claim:
+  receiver: address
+  amount: uint256
+
+event ClaimMinted:
+  claim: Claim
+
+@external
+def __init__(merkle_root: bytes32):
+    self.merkle_root = merkle_root
+
+@external
+def mint(proof: DynArray[bytes32, 1024]):
```

Comment:
Would be nice to claim on behalf of other people. Also proof should only be 20 nodes deep (log_2(160) where 160 is number of bytes in an address
```suggestion
def mint(proof: DynArray[bytes32, 20], receiver: address=msg.sender):
```